editor/src/components/navigator/navigator-item/navigator-item-dnd-container.tsx

```diff
@@ -354,14 +379,19 @@ function isHintDisallowed(elementPath: ElementPath | null, metadata: ElementInst
 export const NavigatorItemContainer = React.memo((props: NavigatorItemDragAndDropWrapperProps) => {
   const editorStateRef = useRefEditorState((store) => store.editor)
 
+  const [isDragSessionInProgress, updateDragSessionInProgress] = useAtom(DragSessionInProgressAtom)
```

Comment:
have you tried using `collected.isDragging` from the useDrag hook? https://react-dnd.github.io/react-dnd/docs/api/use-drag#usedrag